package.json

```diff
@@ -32,12 +32,14 @@
     "@lweller/leaflet-areaselect": "^1.3.0",
     "@radiantearth/stac-fields": "1.0.0-beta.13",
     "@radiantearth/stac-migrate": "~1.0.3",
+    "@turf/bbox-polygon": "^6.5.0",
     "axios": "^0.21.4",
     "bootstrap-vue": "^2.21.2",
     "bs58": "^4.0.1",
     "commonmark": "^0.29.3",
     "core-js": "^3.6.5",
     "leaflet": "~1.7.1",
+    "lodash.get": "^4.4.2",
```

Comment:
I'd propose to remove this, see https://github.com/radiantearth/stac-browser/pull/157/files#r889715354